packages/react/src/primitives/Expander/ExpanderItem.tsx

```diff
@@ -0,0 +1,65 @@
+import classNames from 'classnames';
+import { Item, Header, Trigger, Content } from '@radix-ui/react-accordion';
+
+import { IconExpandMore } from '../Icon';
+import { View } from '../View';
+import { Primitive } from '../types/view';
+import { ExpanderItemProps } from '../types/expander';
+import { ComponentClassNames } from '../shared/constants';
+import { splitPrimitiveProps } from '../shared/styleUtils';
+import { useStableId } from '../shared/utils';
+
+export const EXPANDER_ITEM = 'expander-item';
+export const EXPANDER_HEADER = 'expander-header';
+export const EXPANDER_ICON = 'expander-icon';
+// export const EXPANDER_CONTENT = 'expander-content';
+export const EXPANDER_CONTENT_TEXT = 'expander-content-text';
+
+export const ExpanderItem: Primitive<ExpanderItemProps, typeof Item> = ({
+  children,
+  className,
+  title,
+  ..._rest
+}) => {
+  const triggerId = useStableId();
+  const contentId = useStableId();
+  const { rest } = splitPrimitiveProps(_rest);
+  return (
+    <Item
+      className={classNames(ComponentClassNames.ExpanderItem, className)}
+      data-testid={EXPANDER_ITEM}
+      {...rest}
+    >
+      <Header
```

Comment:
Maybe we can set `asChild={true}` later and use our own `Heading` as wrapper, which will allow users to specify the heading level. But I ran into some errors when I tried to do so because of `ref`. Hopefully this could get resolved once we start forwarding `ref`

Reply:
Yeah, maybe wait on supporting custom Header until we forward refs. Want to create a ticket?

Reply:
Creating